api/tests/shared/integration/infrastructure/repositories/membership-repository_test.js

```diff
@@ -69,4 +69,102 @@ describe('Integration | Infrastructure | Repository | membership-repository', fu
       expect(foundMemberships).to.have.lengthOf(0);
     });
   });
+
+  describe('#findByUserIdAndOrganizationId', function () {
+    context('When organization is not required', function () {
+      it('should retrieve membership with given useId and OrganizationId', async function () {
```

Comment:
Le standard de notre équipe au niveau des tests, est de ne plus utiliser le terme _should_.

```suggestion
      it('retrieves membership with given useId and OrganizationId', async function () {
```

Reply:
Je trouve dommage d'avoir des standards non lintable, ça ajoute de la charge cognitive pendant la revue 😢

Reply:
Un article qui parle du sujet https://stackoverflow.blog/2020/07/20/linters-arent-in-your-way-theyre-on-your-side/ 

Le point que je trouve le plus important :

```
The introduction of linters might not only free up time during code review for more important decision making and learning, it might even reduce the number of requests in total. Lint out those that were about style bickery. Standardizing your code is a great way to move the conversation to a more productive level.
```

Reply:
(Je précise que je vais bien sûr prendre en compte le commentaire 😉)